lib/engine-addon.js

```diff
@@ -143,10 +144,12 @@ var buildVendorJSTree = memoize(function buildVendorJSTree(vendorTree) {
 
 var buildVendorCSSTree = memoize(function buildVendorCSSTree(vendorTree) {
   // Filter out the CSS so that we can process it correctly.
-  return new Funnel(vendorTree, {
+  var vendorCSSTree = new Funnel(vendorTree, {
     include: ['**/*.css'],
     exclude: ['vendor/**/*.*']
   });
+
+  return new BroccoliDebug(vendorCSSTree, this._debugLabel + ':vendor-style:input');
```

Comment:
Good point! I actually built something into broccoli-conditional-debug for this in https://github.com/rwjblue/broccoli-conditional-debug/pull/1.